backend/cfg/linear_to_cfg.ml

```diff
@@ -318,8 +315,8 @@ let check_and_register_traps t =
      then it has a registered exn successor or interproc exn. *)
   let f _ (block : C.basic_block) =
     let n = Label.Set.cardinal block.exns in
-    assert ((not block.can_raise_interproc) || block.can_raise);
-    assert ((not block.can_raise) || n > 0 || block.can_raise_interproc)
+    assert ((not (Cfg.can_raise_interproc block)) || block.can_raise);
```

Comment:
This is not needed any more, it just checks the invariant that `block.can_raise_interproc` implies `block.can_raise`.